Automattic/wp-calypso
# Test: Refactor lib/features-list test to use fake dom

Follow up for #4294.

Refactors lib/features-list test to use fake dom helper.
I also ES6fied test while cleaning file from ESLint warnings.

```
$ npm run test-client

  FeaturesList
    initialize
      ✓ should populate the list of features
      ✓ should set attributes properly

  2 passing (181ms)

```

#### Testing instructions
 
1. Run `git checkout update/refactor-features-list-tests`
2. Execute `npm run test-client` and check that all tests for the module pass
